langfuse/src/langfuse.ts

```diff
@@ -24,10 +25,13 @@ export class Langfuse extends LangfuseCore {
 
   constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
```

Comment:
I think they need to be made nullable so that the user does not need to set them. I added a test.